src/utils/get-pack-nr1-url.js

```diff
@@ -1,16 +1,22 @@
+import {
+  NR1_PACK_DETAILS_NERDLET,
+  NR1_GUIDED_INSTALL_NERDLET,
+  NR1_EXPLORER_NERDLET,
+} from '../data/constants';
+
 // FIXME: update this to production URL when deployed / launched
 const NR1_BASE_URL = 'https://staging-one.newrelic.com';
 const NR1_BASE_URL_LOCAL = 'https://dev-one.newrelic.com';
 const NERDLET_PATH = `launcher/nr1-core.explorer/`;
 
 /**
- * @param {string} packId The ID for an observability pack.
+ * @param {string} quickstartId The ID for an observability pack.
  * @param {boolean} [debug] If set to true, this will add `packages=local`.
  * @returns {string} The URL for the pack details within the platform.
  */
-const getPackNr1Url = (quickstartId, nerdletId, debug = false) => {
+export const getPackNr1Url = (quickstartId, debug = false) => {
```

Comment:
This file had to be adjusted to run on Node.js as well, could you change this to a module export?
```suggestion
const getPackNr1Url = (quickstartId, debug = false) => {
```